mesheryctl/internal/cli/root/config/config_test.go

```diff
@@ -237,3 +240,171 @@ func TestSetComponents(t *testing.T) {
 		}
 	}
 }
+
+// TODO: Shift Testing utility functions to meshkit so import cycle problems can be eliminated in future
+
+// func TestChangePlatform(t *testing.T) {
+// 	type args struct {
+// 		contextName string
+// 		platform    string
+// 	}
+
+// 	currDir := utils.GetBasePath(t)
+// 	fixtureDir := currDir + "/fixtures"
+// 	testConfigPath := fixtureDir + "/TestConfig.yaml"
+
+// 	// Read and write to the test config file
+// 	utils.SetupCustomContextEnv(t, testConfigPath)
+
+// 	mctlCfg, _ := GetMesheryCtl(viper.GetViper())
+
+// 	tests := []struct {
+// 		name    string
+// 		args    args
+// 		wantErr bool
+// 		golden  string
+// 	}{
+// 		{
+// 			name:    "Update platform in gke context (valid context)",
+// 			args:    args{contextName: "gke", platform: "testplatform"},
+// 			wantErr: false,
+// 			golden:  "changeplatform.valid.golden",
```

Comment:
when this code moves to meshkit, all these golden files will also be moved to meshkit, right?

Reply:
Nope, actually testing functions needs to be shifted to a sub-package under utils, to avoid import-cycle issues, I'll address those changes in some other PR.